perun-cli/Perun/Agent.pm

```diff
@@ -64,6 +64,16 @@ sub gotMfaPrivilegeException;
 sub new {
 	my $self = fields::new(shift);
 
+	$self->{_useNon} = shift;
+	if ($self->{_useNon}) {
+		if (defined($ENV{PERUN_OIDC}) && $ENV{PERUN_OIDC} eq "1") {
```

Comment:
can we use the oidc's perun url in this case (from the config file, same as would be used in other scripts)?